tox.ini

```diff
@@ -57,6 +59,7 @@ commands =
     mypy-focal: mypy --python-version 3.7 uaclient/ features/ lib/
     black: black --check --diff uaclient/ features/ lib/ setup.py
     isort: isort --check --diff uaclient/ features/ lib/ setup.py
+    shellcheck: bash -c "shellcheck -S warning -C always tools/{*.sh,make-release,make-tarball} debian/ubuntu-advantage-tools.{config,postinst,postrm,prerm} lib/*.sh sru/release-27{,.3,.5,.8}/*.sh update-motd.d/**"
```

Comment:
Given that none of the scripts under `sru/` is shellcheck clean, should we just exclude it for now? Matching something like, with a couple of other minor changes:
```
tools/{*.sh,make-release,make-tarball} debian/*.{config,postinst,postrm,prerm} lib/*.sh update-motd.d/*
```
Also I'd remove `-C always` which may not play well with logging in automation.

Reply:
I will update the sru/ folder selection to include only the latest release i.e. `27.8` currently.

Reply:
`-C always` was just a way of testing the options, so lemme remove it, esp. if it will be a possible issue in the future.